Source/utils/file_util.cpp

```diff
@@ -111,6 +111,8 @@ bool ResizeFile(const char *path, std::uintmax_t size)
 	}
 	::CloseHandle(file);
 	return true;
+#elif defined(__PSP__)
+	return false;

```

Comment:
Some form of `ResizeFile` is unfortunately required for save games to work

Reply:
While it would obviously require more code, the PSP has its own save data utility accessible to all games. It is fairly trivial to implement, you can either call the SceUtility functions when saving the game or implement the save data dialog and let players choose files so they can scroll through the fancy icons ;)

https://github.com/pspdev/pspsdk/blob/master/src/samples/savedata/utility/main.c

Reply:
PSP libc is supposed to already support truncate, not sure what's up with it missing

https://github.com/pspdev/newlib/issues/1
https://github.com/pspdev/newlib/issues/2

Reply:
It's the problem on https://github.com/pspdev/psptoolchain/issues/108. C++ for some reason doesn't have the functions C does.